app/assets/stylesheets/application.scss

```diff
@@ -12,3 +12,4 @@
  */
 
 @import 'bootstrap';
+@import 'announcements.scss'
```

Comment:
No need the `.scss`

Reply:
But when I don't do it, it doesn't reflect on my takoyaki.

Reply:
Oooh interesting. Okay then